domains/kikidevelopment.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "imdakiki",
+           "email": "skullkid73@outlook.com",
+           "discord": "898255050592366642"
+        },
+    
+        "record": {
+            "TXT": "d72b33b3518595dd14fe63bdad314b"
```

Comment:
TXT doesnt really do anything, if you want to connect your website to domain this is not the way to do it. If you dont have a website you can either use Github Pages or Cloudflare Pages

Reply:
I'm trying to do the Github pages but it says as a TXT record

Reply:
oh thats not how you do it, you can follow along here https://www.is-a.dev/docs/github_pages/